src/en/controllers-ha.md

```diff
@@ -15,19 +15,19 @@ Juju HA is tightly integrated with the MongoDB database since that is where all
 environment data is stored. The MongoDB software has a native HA implementation
 and this is what Juju HA uses. A MongoDB cluster is called a
 [replica set](http://docs.mongodb.org/manual/replication/) and, in the context
-of Juju HA, i) each of its members corresponds to a different Juju state server
-and ii) the MongoDB replica set master corresponds to the Juju state server master.
+of Juju HA, i) each of its members corresponds to a different controller
```

Comment:
I don't think we should use this method of labelling. Consider re-doing as a list